extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsMiddleware/FunctionsHttpProxyingMiddleware.cs

```diff
@@ -27,6 +30,13 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
 
             await next(context);
 
+            if (context.GetInvocationResult()?.Value is IActionResult actionResult)
+            {
+                ActionContext actionContext = new ActionContext(httpContext, httpContext.GetRouteData(), new ActionDescriptor());
```

Comment:
Do we have plans to fill out `ActionDescriptor`?

Reply:
Discussed at sync yesterday and no, we do not. Looking at [similar code](https://github.com/Azure/azure-functions-host/blob/76591f722931f651caddeb0f6080d934545246dd/src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs#L73) in the host side and we do not fill it out either.

Reply:
If we don't do it in host code I am fine leaving this empty then.

I do wonder what functionality we are excluding here? `ActionDescriptor` essentially holds a breakdown the MVC endpoint that is being ran. We should consider populating it in the future.

Reply:
There's no action associated with the context and we have a terminating middleware, no action selection will be performed. As @satvu pointed out, that's the same pattern used by the host.